### PR TITLE
[dev] Documentation Components & DataGrid JSS

### PR DESCRIPTION
**Pull request by morethanfire**

### CODE REVIEW

File: src/dataDisplay/dataGrid/dataGridTableRow.jsx [hunk 1]
```diff
@@ -140,7 +154,9 @@ class DataGridTableRow extends React.Component {
                                         />
                                     </Grid.Column>
 
-                                    <Grid.Column>
+                                    <Grid.Column
+                                        className={classes.accessorGridColumn}
+                                    >
                                         {accessor}
```
Comment (groberts314) [inline src/dataDisplay/dataGrid/dataGridTableRow.jsx:160]: "accessor" as name refers to the function or means of accessing something.
So, in my opinion, `accessor` is not a good identifier here for the data or value that has been accessed via the provided accessor (and thus `accessorGridColumn` is not a good class name identifier).

Could we rename these?  The name should indicate that this is the data or value to be displayed in the column/cell.
Maybe:
`accessor` => `accessedDataValue`
`classes.accessorGridColumn` => `classes.dataValueGridColumn`

Or something along those lines?
Possibly NAB, but this just stuck out at me, and I had to think a bit and look at other code.

File: docs/src/inputs/radio/radio.jsx [hunk 2]
```diff
@@ -443,15 +453,20 @@ export default class ElementsRadio extends React.Component {
...
+                    >
+                        <Radio.Item id="block_name--element_name-1" label="Option 01" />
+                        <Radio.Item id="block_name--element_name-2" label="Option 02" />
+                        <Radio.Item id="block_name--element_name-3" label="Option 03" />
                     </Radio><br /><br />
 
                     <Radio
                         checked={this.state.checkedPill}
-                        name="pill-onchange-sample"
+                        name="pillOnchangeSample"
```
[the remark this answers is not in the document]
Comment (morethanfire) [reply docs/src/inputs/radio/radio.jsx:469]: Yeah... :/
Fixing...

File: src/dataDisplay/dataGrid/dataGridTable.jsx [hunk 3]
```diff
@@ -129,17 +128,17 @@ class DataGridTable extends React.PureComponent {
         });
 
         let tableBody;
-        const bodyClasses = ClassNames({ [`${classNamePrefix}_drop_shadow`]: dropShadow });
+        const bodyClasses = ClassNames({ [`${BEM_DATA_GRID}--table_body-drop_shadow`]: dropShadow });
 
-        if (sortable === true) {
+        if (isArray(sortable) && sortable.length === 1 && !isArray(dataProp[0])) {
```
[the remark this answers is not in the document]
Comment (morethanfire) [reply src/dataDisplay/dataGrid/dataGridTable.jsx:133]: We were never using `sortable` as a boolean and never will. So this is was to actually fix a bug that was discovered along the way.

File: src/dataDisplay/dataGrid/dataGridTableRow.jsx (hunk 1 above)
Comment (morethanfire) [reply src/dataDisplay/dataGrid/dataGridTableRow.jsx:160]: Sure...

Fixing.

Comment (morethanfire) [reply src/dataDisplay/dataGrid/dataGridTableRow.jsx]: Fixed.

File: docs/src/inputs/radio/radio.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (morethanfire) [reply docs/src/inputs/radio/radio.jsx]: Fixed.

[the remark this answers is not in the document]
Comment (groberts314) [reply docs/src/inputs/radio/radio.jsx]: Looks good.  Resolving.

File: src/dataDisplay/dataGrid/dataGridTable.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (groberts314) [reply src/dataDisplay/dataGrid/dataGridTable.jsx:133]: Sounds good.  We'll go with this for now.  If we run into issues with consumers not knowing how to configure `sortable` we'll maybe iterate on this (add console warnings, more documentation, whatever we need).  Resolving.

File: src/dataDisplay/dataGrid/dataGridTableRow.jsx (hunk 1 above)
Comment (groberts314) [reply src/dataDisplay/dataGrid/dataGridTableRow.jsx]: I'll live with `accessorDataValue` :grin:
I would have preferred `accessedDataValue` (i.e. we used the accessor and have _accessed_ it, and here it is!)
But meh ... no biggie.
Resolving.

**Comment by morethanfire** [comment]
Checks passed, PR has an approval, and we are all waiting on this to be merged. :shipit:

**Pull request merged**